packages/create-bison-app/index.js

```diff
@@ -66,6 +66,8 @@ module.exports = async ({ name, ...answers }) => {
         variables.host.createAppsAndPipelines,
       task: async () => {
         const repoName = variables.githubRepo.match(/(\w+\/\w+).git$/)[1];
+        // ! Bug w/ Pipelines, better error handling needed.
+        // await fsPromises.writeFile("Procfile", "web: yarn db:deploy && yarn start");
```

Comment:
Should we add a follow up issue to track this?

Reply:
I'll get this added as a follow-up.